ragstack/colbert/token_embedding.py

```diff
@@ -43,26 +41,32 @@ class PassageEmbeddings:
     __token_embeddings: List[PerTokenEmbeddings]
     __text: str
     __title: str
-    __id: uuid.UUID
+    __id: str
+    __model: str
+    __dim: int
 
     def __init__(
         self,
         text: str,
-        title: str = "",
-        part: int = 0,
-        id: uuid.UUID = None,
+        title: str = "",  # keep this as backward compatibility, use id instead
```

Comment:
do we need to worry about backward compatibility? this hasn't been released yet, correct?

Reply:
nope